repeater/src/main/scala/com.snowplowanalytics.snowplow.storage.bigquery.repeater/Flow.scala

```diff
@@ -81,8 +81,6 @@ object Flow {
                                         (event: Model.Record[F, EventContainer]): F[Either[BadRow, Unit]] = {
     val res = for {
       ready <- EitherT.right(event.value.isReady(backoffTime))
-      reconstructedEvent <- EitherT(event.value.parsePayload)
-        .leftFlatMap { e => EitherT(event.ack.as(e.asLeft[PayloadParser.ReconstructedEvent])) }
```

Comment:
This was just a residue of a previous version?

Reply:
I think so (though its a bit of mystery). Probably just an artifact from some pre-`snowplow-badrows` RC.